src/ui/pages/inventory/index.tsx

```diff
@@ -10,13 +9,10 @@ import { typography } from '../../../mn-constants';
 import styled from '../../themes/styled';
 
 export interface Props {
-  userId: string;
-  inventory: EconomicResource[] | [];
-  error?: ApolloError | undefined;
-  loading: boolean;
+  inventory: any;
```

Comment:
You can use the `EconomicResourcesFilteredQuery` type from `HOC/pages/inventory/InventoryPage.generated`.

```suggestion
  inventory: EconomicResourcesFilteredQuery['economicResourcesFiltered'];
```

This way you can skip the `inventory as EconomicResource[]` and get the proper type for your item.